src/intl/en/glossary.json

```
@@ -223,6 +223,8 @@
   "liquidity-definition": "Liquidity is how quickly and easily an asset can be converted into cash or another asset. Decentralized exchanges like Uniswap have multiple liquidity pools where asset holders can deposit their assets where traders can buy and sell them in a decentralized way in exchange for rewards.",
   "liquidity-tokens-term": "Liquidity tokens",
   "liquidity-tokens-definition": "Liquidity tokens (LST) are digital tokens issued to participants who deposit assets into a liquidity pool, which is a collection of funds locked in a smart contract and used to facilitate trading on a decentralized exchange (DEX).<br />These tokens represent the participant's share of the pool and can be redeemed later for the initial deposit plus a portion of the trading fees generated by the pool's activity. Essentially, liquidity tokens serve as a proof of ownership or stake in a liquidity pool, allowing holders to earn rewards while providing the necessary liquidity for others to trade different cryptocurrency pairs efficiently.",
+  "liquid-staking-tokens-term": "Liquid staking tokens",
+  "liquid-staking-tokens-definition": "A derivative token representing that represents the locked cryptocurrency which user gets by staking a token. When a user stakes their assets, they receive an equivalent amount of Liquid Staking Tokens. These LSTs can then be traded, sold, or used in other DeFi protocols, providing liquidity to the staker even while their original assets remain staked.",
```

Looks great, just going to adjust a bit

```suggestion
  "liquid-staking-tokens-definition": "A derivative token that represents ownership of the locked cryptocurrency that a user is staking. Upon staking an asset, some platforms allowing minting of liquid staking tokens (LSTs), which represent an equivalent share of the locked tokens. These LSTs can then be traded, sold, or used in other DeFi protocols, improving capital efficiency for the staker by allowing access to liquidity from their funds, even while their original assets remain staked.",
```